docs/operate-and-deploy/installation/server-config/config-reference.md

```diff
@@ -612,11 +612,13 @@ property has the value `KSQL_PROCESSING_LOG`.
 Toggles whether or not the processing log should include rows in log
 messages. By default, this property has the value `false`.
 
-### ksql.logging.server.skipped.response.codes
+### ksql.logging.server.rate.limited.response.codes
 
-A comma-separated list of HTTP response codes to skip during server
-request logging. This is useful for ignoring certain 4XX errors that you
-might not want to show up in the logs.
+A list of `path:rate_limit` pairs, to limit the rate of server request
```

Comment:
Clarify the units of the rate limit (and also for the other config below). Appears to be number of messages per second?